test/js/simple-length.js

```diff
@@ -90,4 +90,24 @@ suite('SimpleLength', function() {
     assert.strictEqual(lengthAddition.value, 5);
   });
 
+  test('convertToCalcLength method returns a CalcLength object with a single length value type when called from a SimpleLength object', function() {
```

Comment:
We already know it's being called from a SimpleLength object since we're in the SimpleLength test suite. How about

"asCalcLength method returns a CalcLength with single value"?